pyiron_base/state/install.py

```diff
@@ -111,7 +115,7 @@ def install_dialog():
                 config_file_name="~/.pyiron",
                 zip_file="resources.tar.gz",
                 resource_directory="~/pyiron/resources",
-                giturl_for_zip_file="https://github.com/pyiron/pyiron-resources/releases/download/0.0.17/resources-0.0.17.tar.gz",
+                giturl_for_zip_file="https://github.com/pyiron/pyiron-resources/releases/download/0.0.18/resources-0.0.18.tar.gz",
```

Comment:
```suggestion
                giturl_for_zip_file="https://github.com/pyiron/pyiron-resources/releases/download/latest/resources-0.0.18.tar.gz",
```
I just saw [here](https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases) that we can link to the latest release, I guess that makes sense by default.

Reply:
Actually I think it's better to not pass any arguments here and just use the defaults defined in the `install_pyiron` function.